kernel/Kconfig

```diff
@@ -479,8 +479,13 @@ config NUM_PIPE_ASYNC_MSGS
 	  Setting this option to 0 disables support for asynchronous
 	  pipe messages.
 
+config MEMPOOL
+	bool "Memory pools"
+	default y
+
```

Comment:
Please add a help section for this config

Reply:
done